persistence/src/test/java/de/cyface/persistence/DatabaseHelperTest.java

```diff
@@ -112,7 +112,41 @@ public void tearDown() {
     }
 
     /**
-     * Test that loading the EventTable which was introduced in the Database V12 Upgrade works.
+     * Test that upgrading the {@link MeasurementTable} to Database V13 does not loose entries.
+     */
+    @Test
+    public void testMigrationV12ToV13() {
+
+        // Arrange
+        // This is simpler than copying and adjusting the code from previous versions
+        createV11DatabaseWithData(db);
+        oocut.onUpgrade(db, 11, 12);
+
+        // Act - This is how the method is called by the system (not incrementally!)
```

Comment:
That's just a reminder for future tests that the system calls this method not incrementally, i.e. if we want to rest V10->V13 migration, we should call (db, 10, 13) to ensure we test the code realistically.

In this case we actually just want to test V12-V13, the V11-V12 migration before is part of the "Arrange". We avoid copy/pasting a sample dataset for V12 this way by reusing the V11 sample data set.